### PR TITLE
Finish comments likes and pagination

### PR DESCRIPTION
**Pull request by AleksandrRazhev**

### CODE REVIEW

File: src/app/api/graphql/route.ts [hunk 1]
```diff
@@ -119,8 +119,12 @@ const resolvers = {
             await CommentModel.updateOne({ _id: id }, comment);
             return await CommentModel.findById(id);
         },
-        deleteComment: async (parent: any, { id }: { id: string }) => {
-            return await CommentModel.deleteOne({ _id: id });
+        deleteComment: async (parent: any, { commentId, userId }: { commentId: string; userId: string }) => {
+            const currentComment = await CommentModel.findById(commentId);
+            if (!currentComment) return;
```
Comment (alexbaumgertner) [inline src/app/api/graphql/route.ts:124]: Тут хорошо бы отправлять HTTP статусы 404 для не найденного комментария и 403 если нет прав
https://www.apollographql.com/docs/apollo-server/data/errors/#setting-http-status-code-and-headers

Comment (alexbaumgertner) [reply src/app/api/graphql/route.ts:124]: Screenshot 2024-09-21 at 15 13 52
Screenshot 2024-09-21 at 15 13 57
я пробую варианты

Comment (alexbaumgertner) [reply src/app/api/graphql/route.ts:124]: документация рекомендует в Header 200 всегда отправлять :)

> While Apollo Server does enable you to set HTTP status codes based on errors thrown by resolvers, best practices for GraphQL over HTTP encourage sending 200 whenever an operation executes. So, we don't recommend using this mechanism in resolvers, just in the context function or in a plugin hooking into an early stage of the request pipeline.

**Review by zladen — APPROVED** [review]

**Review by alexbaumgertner — APPROVED** [review]

**Review by alexbaumgertner — APPROVED** [review]

**Review by alexbaumgertner — APPROVED** [review]

**Review by alexbaumgertner — APPROVED** [review]

File: src/app/api/graphql/route.ts [hunk 2]
```diff
@@ -48,10 +48,10 @@ const resolvers = {
             return TripModel.findById(id);
         },
 
-        comments: async (parent: any, { event_id }: { event_id: string }) => {
+        comments: async (parent: any, { event_id, limit }: { event_id: string; limit?: number }) => {
```
Comment (alexbaumgertner) [inline src/app/api/graphql/route.ts:51]: Пример комментария

**Pull request merged**